src/plugins/data/public/search/aggs/buckets/lib/time_buckets/time_buckets.ts

```diff
@@ -29,9 +29,9 @@ import {
   EsInterval,
 } from './calc_es_interval';
 
-interface Bounds {
-  min: Date | number | null;
-  max: Date | number | null;
+export interface Bounds {
+  min: Moment | Date | number | null;
```

Comment:
@VladLasitsa  not sure about it, we already have the TimeRangeBounds interface.

export interface TimeRangeBounds {
  min: Moment | undefined;
  max: Moment | undefined;
}

Reply:
So, I think we can use TimeRangeBounds instead of Bounds. I removed Bounds interface and start using TimeRangeBounds